pkg/commands/volume.go

```diff
@@ -48,15 +48,15 @@ func (v *VolumeCommand) ExecuteCommand(config *v1.Config, buildArgs *dockerfile.
 	for _, volume := range resolvedVolumes {
 		var x struct{}
 		existingVolumes[volume] = x
-		err := util.AddPathToVolumeWhitelist(volume)
+		err := util.AddVolumePathToWhitelist(volume)
 		if err != nil {
 			return err
 		}
 
 		// Only create and snapshot the dir if it didn't exist already
 		if _, err := os.Stat(volume); os.IsNotExist(err) {
 			logrus.Infof("Creating directory %s", volume)
-			v.snapshotFiles = []string{volume}
+			v.snapshotFiles = append(v.snapshotFiles, volume)
```

Comment:
Just a comment about this change. This is a bug I found where it is not respecting multiple volumes created by the same `VOLUME` directive.

The `Dockerfile_test_volume` test did not catch it because `tmp` is already a directory and so `v.snapshotFiles` did not get overridden. I added an extra volume for the directive to create to provide test coverage.
```
VOLUME /foo/bar /tmp /qux/quux
```